utils/updateBooks.js

```diff
@@ -1,19 +1,19 @@
-var moment = require('moment-timezone');
-var Batch = require('./Batch');
+import moment from 'moment-timezone';
+import Batch from './Batch';
 
-var updateBooks = new Batch({
+const updateBooks = new Batch({
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).